jvm-packages/create_jni.py

```diff
@@ -7,6 +7,10 @@
 import sys
 from contextlib import contextmanager
 
+# usage
+# ./create_jni.py [ON|OFF]
+#  ON : USE_CUDA=ON
+#  OFF: USE_CUDA=OFF
```

Comment:
Let's use `argparse` Python module to provide a proper CLI interface:
```
python create_jni.py --use-cuda ON
python create_jni.py --use-cuda OFF
python create_jni.py  # CUDA is off by default
```

If you'd like, I can push to this branch.

Reply:
Hi @hcho3, Pls help with that.

Reply:
I pushed a new commit.